cmake/FindProtobuf-c.cmake

```diff
@@ -0,0 +1,86 @@
+#  PROTOBUF-C_FOUND - System has libyang
```

Comment:
Why Libyan?!

Reply:
i pickup from them and i would like to not modify it since it seems to work that way. Best effort for the moment, no need to clean stuff that could be change very soon.

Reply:
This file is apache licensed... @pgrandin what do you think?

Reply:
Mixing ASL with our GPL2 is tricky indeed. We need to look if we already did it somewhere, and we need to check what's defined as linking (as this seems to be a way out of the issue).

https://opensource.stackexchange.com/questions/1357/can-i-link-a-apache-2-0-library-into-software-under-gplv2

Reply:
The entire protobuf this patch removes was ASL. This is one file used only at build time.

Reply:
according to the Doc cmake itself has FindProtobuf: https://cmake.org/cmake/help/v3.0/module/FindProtobuf.html

Reply:
Okay, the support is only for c++ compile, not for c, so we need to rewrite this probably. Hopefully we can reuse some of the cmake sources.